src/plugins/auto/multi_executable_network.cpp

```diff
@@ -136,8 +136,8 @@ IE::Parameter MultiExecutableNetwork::GetMetric(const std::string& name) const {
         IE_SET_METRIC_RETURN(SUPPORTED_CONFIG_KEYS, configKeys);
     } else if (name == ov::execution_devices) {
         std::vector<std::string> exeDevices = {};
-        for (auto n : _multiSContext->_devicePriorities) {
-            exeDevices.push_back(n.deviceName);
+        for (auto&& n : _multiSContext->_networksPerDevice) {
+            exeDevices.push_back(n.first);
```

Comment:
the network loading order is undefined, but we are expecting a defined order here, like GPU,CPU, the _networkPerDevice may be go with CPU,GPU

Reply:
It maybe has some device that failed to loadnetwork in  _devicePriorities, so replace _devicePriorities with _networkPerDevice , maybe we can add a sort here?